### PR TITLE
Add LambdaTest integration (from #2424)

### PR DESCRIPTION
**Pull request by mateuszwojtczak**
This PR is re-created from #2424

### CODE REVIEW

File: docs/integrations/lambdatest.mdx
Comment (pdenert) [inline docs/integrations/lambdatest.mdx:1]: You can remove this. This is only needed for old documentation pages that have been moved to new locations during documentation changes.

File: docs/documentation/integrations/lambdatest.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+To run Android UI tests on LambdaTest:
+
+1. Upload the app under test APK to LambdaTest ([see docs][LT_app_docs])
+1. Upload the instrumentation app APK to LambdaTest ([see docs][LT_test_docs])
+1. Start test execution on LambdaTest ([see docs][LT_execute_docs])
+
+
+
+
+
```
Comment (pdenert) [inline docs/documentation/integrations/lambdatest.mdx:41]: Why so many empty lines?

Comment (mateuszwojtczak) [reply docs/documentation/integrations/lambdatest.mdx:41]: That's a good question :)

**Review by pdenert — APPROVED** [review]
lgtm

**Pull request merged**